core/utils/string.js

```diff
@@ -135,44 +138,44 @@ Blockly.utils.string.wrap = function(text, limit) {
  * @return {string} Wrapped text.
  * @private
```

Comment:
Based on the style guide, I don't think this should be marked private. It just doesn't get exported. However, we can wait for Chris to confirm that.

Reply:
Updated since this has been long since confirmed, and also updated the exports style to reflect our current convention.